specification.scs

```diff
@@ -0,0 +1,26 @@
+sc_web_specification
+    <- concept_reusable_component_specification;;
+
+sc_web_specification = [*
+sc_web
+    <- concept_reusable_component;
+    <- concept_atomic_reusable_component;
+    <- concept_independent_reusable_component;
+    <- concept_reusable_interface_component;
+    <- concept_reusable_dynamically_installed_component;
+    <- concept_reusable_source_code_component;
+
+    => nrel_sc_identifier: [SC-web specification](* <- lang_en;; *);
+    => nrel_key_sc_element: concept_animal;
+    => nrel_purpose: [SC-web specification purpose](* <- lang_en;; *);
+    => nrel_explanation: [SC-web explanation] (*<- lang_en;; *);
+    => nrel_authors: ... (* <- empty_set *);
+    => nrel_component_dependencies: ... (* <- empty_set;; *);;
+
+concept_dog
+    => nrel_install_scripts: [cd scripts && ./install_deps_ubuntu.sh](* <- lang_en;; *);
+    => nrel_install_scripts: [cd scripts && ./install.sh](* <- lang_en;; *);
```

Comment:
I think there shouldn't be English language. Probably there should be script language?